package.json

```diff
@@ -94,7 +94,7 @@
     "@hashicorp/react-button": "^6.0.3",
     "@hashicorp/react-call-to-action": "^4.1.1",
     "@hashicorp/react-checkbox-input": "^5.0.1",
-    "@hashicorp/react-code-block": "^4.5.0",
+    "@hashicorp/react-code-block": "5.1.0-canary-20220518102202",
```

Comment:
Note: this is a pre-release of https://github.com/hashicorp/react-components/pull/591. Intention is to merge the `react-components` PR first, get a stable release out, and then bump to stable here before merging.